mlir/lib/IR/AsmPrinter.cpp

```diff
@@ -1073,6 +1073,37 @@ static StringRef sanitizeIdentifier(StringRef name, SmallString<16> &buffer,
   return name;
 }
 
+unsigned AliasInitializer::uniqueAliasNameIndex(
+    StringRef alias, llvm::StringMap<unsigned> &nameCounts,
+    llvm::StringSet<llvm::BumpPtrAllocator &> &usedAliases) {
+  // Get nameIndex that will not generate conflicting name.
+  unsigned nameIndex = 0;
+  if (!usedAliases.count(alias)) {
+    usedAliases.insert(alias);
```

Comment:
Can you early exit from this if and drop the else?

Reply:
done